idd/Energy+.idd.in

```diff
@@ -16134,6 +16134,27 @@ SurfaceProperty:HeatTransferAlgorithm:Construction,
        \type object-list
        \object-list ConstructionNames
 
+SurfaceProperty:HeatBalanceSourceTerm,
+       \memo Allows additional heat source term of the surface boundary condition in the surface heat balance calculation
+       \memo For Advanced/Research Usage
+       \min-fields 3
+  A1 , \field Surface Name
+       \required-field
+       \type object-list
+       \object-list SurfaceNames
+  A2 , \field Heat Source Location
+       \type choice
+       \key Outside
+       \key Inside
+       \default Outside
+       \note This field indicates where the heat source applies.
```

Comment:
where the heat source **is applied**.

Reply:
The PR title and description only mention an exterior source term, but it looks like the implementation allows for either inside or outside? Please update these to match.